.github/workflows/ci-testing-deploy.yml

```diff
@@ -15,37 +15,23 @@ on:
       - "*"
     tags-ignore:
       - "*"
-    paths:
-      - "!.vscode/**"
-      - "!*.md"
-      - "!*.png"
-      - "!*.svg"
-      - "!docs/**"
-      - ".github/workflows/ci-testing-deploy.yml"
-      - "**.js"
-      - "**.json"
-      - "**.Makefile"
-      - "**.py"
-      - "**/requirements/*.txt"
-      - "**/services/docker-compose*"
-      - "ci/**"
   pull_request:
     branches:
       - "*"
-    paths:
-      - "!*.md"
-      - "!*.png"
-      - "!*.svg"
-      - "!docs/**"
-      - "!.vscode/**"
-      - "!.vscode-template/**"
-      - "**.py"
-      - "**.js"
-      - "**/requirements/*.txt"
-      - "**.json"
-      - "**.Makefile"
-      - ".github/workflows/ci-testing-deploy.yml"
-      - "ci/**"
+
+  workflow_dispatch:
+    # inputs:
```

Comment:
these comments?

Reply:
uncommented. this should allow for manual triggering